test/asserts.js

```diff
@@ -0,0 +1,15 @@
+import assert from "assert";
+
+export default function(actual, expected) {
+  assert.strictEqual(normalizePath(actual + ""), normalizePath(expected + ""));
+}
```

Comment:
I would prefer a named export (since this file is simply named “asserts”).

```suggestion
export function assertPartEqual(actual, expected) {
  assert.strictEqual(normalizePath(actual + ""), normalizePath(expected + ""));
}
```